lib/commands/preview.ts

```diff
@@ -1,16 +1,24 @@
+import { DEVICE_LOG_EVENT_NAME } from "../common/constants";
```

Comment:
Isn't it better to add the DEVICE_LOG_EVENT_NAME string in the previewAppLogProvider itself instead of using the constants. In this way it will be also available in the public API and we will not require a hardcoded string (`deviceLogData`). Also the `data` suffix in the event name is a little bit strange. It could be just "deviceLog"